playground/backend/internal/setup_tools/builder/setup_builder_test.go

```diff
@@ -195,10 +231,10 @@ func TestCompiler(t *testing.T) {
 		{
 			// Test case with calling Setup with correct data.
 			// As a result, want to receive an expected compiler builder.
-			name: "Test correct compiler builder",
+			name: "Test correct compiler builder with java sdk",
```

Comment:
Should we do the same test for Go and SCIO?

Reply:
Done